### ISSUE TITLE
Some questions about NAN

### ISSUE THREAD

**Issue by JingY0604** [YirongMao/NAN#3]
hi：    
      thank you for sharing your code of NAN, but, I have some questions about it:     
      1. Is there any other way to get pre-trained model and data?  like google drive link?     
      2. How train list and test list generated？     
      3. why face features are extracted by ResNet34 with only SoftMax loss, not CosoNet best model?
thanks!

**Comment by YirongMao** [comment]
> hi：    
> thank you for sharing your code of NAN, but, I have some questions about it:     
> 1. Is there any other way to get pre-trained model and data?  like google drive link?     
> 2. How train list and test list generated？     
> 3. why face features are extracted by ResNet34 with only SoftMax loss, not CosoNet best model?
> thanks!

Thanks for your interest. The purpose of this project is to reproduce the NAN algorithm, NOT to reach top performance. I follow the published protocol of IJB-A to split training and testing dataset. BTW, Sorry for my late response. If you have any other problem, be free to contact me.

**Comment by JingY0604** [comment]
> > hi：    
> > thank you for sharing your code of NAN, but, I have some questions about it:     
> > 
> > 1. Is there any other way to get pre-trained model and data?  like google drive link?
> > 2. How train list and test list generated？
> > 3. why face features are extracted by ResNet34 with only SoftMax loss, not CosoNet best model?
> >    thanks!
> 
> Thanks for your interest. The purpose of this project is to reproduce the NAN algorithm, NOT to reach top performance. I follow the published protocol of IJB-A to split training and testing dataset. BTW, Sorry for my late response. If you have any other problem, be free to contact me.

Thank you for your answer.  but，I have another question，why fixed the size of image set during training？

> > hi：    
> > thank you for sharing your code of NAN, but, I have some questions about it:     
> > 
> > 1. Is there any other way to get pre-trained model and data?  like google drive link?
> > 2. How train list and test list generated？
> > 3. why face features are extracted by ResNet34 with only SoftMax loss, not CosoNet best model?
> >    thanks!
> 
> Thanks for your interest. The purpose of this project is to reproduce the NAN algorithm, NOT to reach top performance. I follow the published protocol of IJB-A to split training and testing dataset. BTW, Sorry for my late response. If you have any other problem, be free to contact me.

Thank you for your answer.  but，I have another question，why fixed the size of image set with 8 during training？

**Comment by YirongMao** [comment]
> > > hi：    
> > > thank you for sharing your code of NAN, but, I have some questions about it:     
> > > 
> > > 1. Is there any other way to get pre-trained model and data?  like google drive link?
> > > 2. How train list and test list generated？
> > > 3. why face features are extracted by ResNet34 with only SoftMax loss, not CosoNet best model?
> > >    thanks!
> > 
> > 
> > Thanks for your interest. The purpose of this project is to reproduce the NAN algorithm, NOT to reach top performance. I follow the published protocol of IJB-A to split training and testing dataset. BTW, Sorry for my late response. If you have any other problem, be free to contact me.
> 
> Thank you for your answer. but，I have another question，why fixed the size of image set during training？
> 
> > > hi：    
> > > thank you for sharing your code of NAN, but, I have some questions about it:     
> > > 
> > > 1. Is there any other way to get pre-trained model and data?  like google drive link?
> > > 2. How train list and test list generated？
> > > 3. why face features are extracted by ResNet34 with only SoftMax loss, not CosoNet best model?
> > >    thanks!
> > 
> > 
> > Thanks for your interest. The purpose of this project is to reproduce the NAN algorithm, NOT to reach top performance. I follow the published protocol of IJB-A to split training and testing dataset. BTW, Sorry for my late response. If you have any other problem, be free to contact me.
> 
> Thank you for your answer. but，I have another question，why fixed the size of image set with 8 during training？

I tuned the image set size, and found 8 is the best.
